lms/static/js/learner_dashboard/views/course_entitlement_view.js

```diff
@@ -0,0 +1,394 @@
+// This is required for karma testing due to a known issue in Bootstrap-v4: https://github.com/twbs/bootstrap/pull/22888
+// The issue is that bootstrap tries to access Popper's global Popper object which is not initialized on loading
+// from the karma configuration. The next version of bootstrap (>v4.2) will solve this issue.
+// Once this is resolved, we should import bootstrap through require-config.js and main.js (for jasmine testing)
+var defineFn = require || RequireJS.require;  // eslint-disable-line global-require
+var Popper = defineFn(['common/js/vendor/popper']);
+defineFn(['common/js/vendor/bootstrap']);
+
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'underscore',
+        'gettext',
+        'moment',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'js/learner_dashboard/models/course_entitlement_model',
+        'js/learner_dashboard/models/course_card_model',
+        'text!../../../templates/learner_dashboard/course_entitlement.underscore',
+        'text!../../../templates/learner_dashboard/verification_popover.underscore'
+    ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             moment,
+             HtmlUtils,
+             EntitlementModel,
+             CourseCardModel,
+             pageTpl,
+             verificationPopoverTpl
+         ) {
+             return Backbone.View.extend({
+                 tpl: HtmlUtils.template(pageTpl),
+                 verificationTpl: HtmlUtils.template(verificationPopoverTpl),
+
+                 events: {
+                     'change .session-select': 'updateEnrollBtn',
+                     'click .enroll-btn': 'handleEnrollChange',
+                     'keydown .final-confirmation-btn': 'handleVerificationPopoverA11y',
+                     'click .popover-dismiss': function() { this.hideDialog(this.$('.enroll-btn-initial')); }
```

Comment:
The anonymous function here is a bit messy, can you update to create a new function in the View and just call that instead?

Reply:
fixed